pkg/operators/otel-metrics/otel-metrics.go

```diff
@@ -87,6 +89,8 @@ func (m *otelMetricsOperator) Init(globalParams *params.Params) error {
 		return nil
 	}
 
+	// TODO: Validate the listen address to immediately return an error
```

Comment:
Any suggestion how to do it quickly?

Reply:
[ResolveTCPAddr](https://pkg.go.dev/net#ResolveTCPAddr)?

Reply:
I did it manually because `ResolveTCPAddr` parses `<any-string>:1234` as `0.0.0.0:1234` instead of an error.